### ISSUE TITLE
External link icon: add empty alt attribute

### ISSUE THREAD

**Issue by jstrothman** [18F/ux-guide#338]
**Background**
In https://github.com/18F/ux-guide/pull/335 we added an icon to indicate external links.
The external link image needs alternate text. I recommend using "on external website" to provide equivalent information.

Mac VoiceOver just ignores the image. I'm not sure what NVIDIA or JAWS do, but it fails automated accessibility testing with Accessibility Insights for Web. (See screenshot at bottom of issue)

**Acceptance criteria _(we'll know we're done when…)_** 
- [ ] Listening on a screenreader, we hear "link: [link text] on external website"
- [ ] Passes automated accessibility testing such as browser-based Accessibility Insights or WAVE

**Tasks**
- [ ] Add alternate text attribute to the image
